v22.1/aws-dms.md

```diff
@@ -77,27 +88,25 @@ A database migration task, also known as a replication task, controls what data
 ### Step 2.2. Task settings
 
 1. For the **Editing mode** radio button, keep **Wizard** selected.
-1. For the **Target table preparation mode**, select either **Truncate** or **Do nothing**.
+1. For the **Target table preparation mode**, select **Truncate** or **Do nothing**.
     <img src="{{ 'images/v22.1/aws-dms-task-settings.png' | relative_url }}" alt="AWS-DMS-Task-Settings" style="max-width:100%" />
-1. Manually create all schema objects in the target CockroachDB cluster. This step is required in order for the migration to populate data successfully.
-    {{site.data.alerts.callout_info}}
-    All tables must have a primary key associated with them. For more information, see [Primary Key Constraint](primary-key.html).
-    {{site.data.alerts.end}}
+1. Check the **Enable CloudWatch logs** option. We highly recommend this for troubleshooting potential migration issues. 
+1. For the **Target Load**, select **Detailed debug**.
+    <img src="{{ 'images/v22.1/aws-dms-cloudwatch-logs.png' | relative_url }}" alt="AWS-DMS-CloudWatch-Logs" style="max-width:100%" />
 
 {{site.data.alerts.callout_info}}
 **Drop tables on target** is unsupported at this time.
 {{site.data.alerts.end}}
 
-{{site.data.alerts.callout_success}}
-When specifying a range of tables, you must ensure the following before data migration can successfully occur:
-
-- The column names within each table being migrated from the source database to CockroachDB are identical.
-- The column types for the columns within each table being migrated from the source database to CockroachDB are compatible.
-{{site.data.alerts.end}}
+### Step 2.3. Table mappings
 
-{% comment %}In order to use **Drop tables on target**, you must [create a new `dms` user](#create-a-new-dms-user) and ensure [`BatchApplyEnabled`](#batchapplyenabled) is set to `False`. {% endcomment %}
+{{site.data.alerts.callout_info}}
+When specifying a range of tables to migrate, the following aspects of the source and target database schema **must** match:
 
-### Step 2.3. Table mappings
+- Column names must be identical.
+- Column types must be compatible.
+- Column nullability must be identical.
+{{site.data.alerts.end}}
 
 1. For the **Editing mode** radio button, keep **Wizard** selected.
 1. Select **Add new selection rule**.
```

Comment:
I know this isn't part of your changes, but "drop down" is open here, elsewhere it is closed.